specs/protocol/isthmus/derivation.md

```diff
@@ -0,0 +1,25 @@
+# Isthmus L2 Chain Derivation Changes
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Network upgrade automation transactions](#network-upgrade-automation-transactions)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Network upgrade automation transactions
+
+The Isthmus hardfork activation block contains the following transactions, in this order:
+
```

Comment:
This section is underspecified. What are the definitions of the network upgrade transactions? The ecotone specs contain examples of this. This is important because the derivation pipeline has to deterministically create these transactions, ie who is the `from`, what is the `to`, what is the `data`? For the contract deployments, we need finalized bytecode, which will need to come from a release

Reply:
Can you send a link to one of the ecotone specs that contains an example of the network upgrade transaction definitions? 

I don't see a clear format in the derivation spec: https://github.com/ethereum-optimism/specs/blob/main/specs/protocol/ecotone/derivation.md#ecotone-blob-retrieval

Reply:
Looks like they are in the wrong place: https://github.com/ethereum-optimism/specs/blob/87e13cba8f34a68858a334fc2587041b712facfa/specs/protocol/derivation.md#network-upgrade-automation-transactions

I can move them to the correct place